src/Adapter/LegacyContextLoader.php

```diff
@@ -0,0 +1,241 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace CommandLineUtils\Controller {
```

Comment:
I'm not sure it's a good practice two namespaces in the same file.
Not sure PSR4 allowing that without a warning :/

Reply:
It was an idea of https://github.com/PrestaShop/PrestaShop/pull/21125#discussion_r499722110 . It has pros and cons :/ I'm open to alternatives but this DummyController, wherever it is, will be a workaround.

Reply:
I'm more in favor of having a dedicated file per class if it's not related to tests. It will be easier to update, manage, move if needed but if everyone agree, I'm not against :sweat_smile:

Reply:
Fixed by https://github.com/PrestaShop/PrestaShop/pull/21125/commits/c64bb28fba3725e14699262556984c5605017b85